f2/src/forms/ContactInfoFormSchema.js

```diff
@@ -20,6 +20,13 @@ const contactFormOptions = {
     label: <Trans id="contactinfoPage.hasValidationErrors" />,
     errorMessage: <Trans id="contactinfoPage.hasValidationErrors" />,
   },
+  emailORphone: {
```

Comment:
This isn't actually a question on the form.

Reply:
We need to keep this, this is used and referenced  in errorSummary. Otherwise will cause run time error

The other one is not used now, i remove it
"enterContactDetails"

Reply:
This is referenced by errorSummary, we need to keep, if removed, run time error will come

Reply:
I take out this from this block, and use others to get label and message, will be in next submmision.